api/src/main/java/org/openmrs/api/impl/ConceptServiceImpl.java

```diff
@@ -1977,4 +1977,46 @@ public boolean hasAnyConceptAttribute(ConceptAttributeType conceptAttributeType)
 		}
 		return mappedClasses;
 	}
+
+	public Concept getConceptByRef(String conceptRef) throws APIException {
+		if(conceptRef == null) {
+			throw new APIException("null isn`t allowed as acceptable input");
```

Comment:
We possibly don't need this, how are NPEs or such cases handled in the entire Service layer?